### PR TITLE
Webpack: Remove automatic selection of browser version

### LINKED ISSUE #1009: WARNING in PlatformTools.js
what is this warning means and how can I fix this :

> WARNING in ./node_modules/typeorm/platform/PlatformTools.js
26:19-32 Critical dependency: the request of a dependency is an expression
    at CommonJsRequireContextDependency.getWarnings (M:\my-app\node_modules\webpac
k\lib\dependencies\CommonJsRequireContextDependency.js:27:4)
    at Compilation.reportDependencyErrorsAndWarnings (M:\my-app\node_modules\webpa
ck\lib\Compilation.js:694:24)
    at Compilation.finish (M:\my-app\node_modules\webpack\lib\Compilation.js:552:9
)
    at applyPluginsParallel.err (M:\my-app\node_modules\webpack\lib\Compiler.js:51
2:17)
    at M:\my-app\node_modules\tapable\lib\Tapable.js:289:11
    at _addModuleChain (M:\my-app\node_modules\webpack\lib\Compilation.js:498:11)
    at processModuleDependencies.err (M:\my-app\node_modules\webpack\lib\Compilati
on.js:468:14)
    at _combinedTickCallback (internal/process/next_tick.js:73:7)
    at process._tickCallback (internal/process/next_tick.js:104:9)

### PR DESCRIPTION
**Pull request by daniel-lang**
This will finally support node projects that use webpack. I also put together the needed changed to make it work in Ionic over at the [example](https://github.com/typeorm/ionic-example#typeorm--018).

fixes #1009

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ however since API is already quite stable we don't expect too much breaking chan
 If we missed a note on some change or you have a questions on migrating from old version, 
 feel free to ask us and community.
 
+## 0.1.8 (future)
```
Comment (pleerock) [inline CHANGELOG.md]: Don't you think we shall provide more details on site as well?

Comment (daniel-lang) [reply CHANGELOG.md]: Might be a good idea, by where? FAQ?

Comment (pleerock) [reply CHANGELOG.md]: not sure, shouldn't it be in http://typeorm.io/#/supported-platforms ?

Comment (pleerock) [reply CHANGELOG.md]: btw this will land in 0.1.7 which Im planning to release later today

**Pull request merged**